src/modules/Providers/views/create/components/ProviderCreateForm.tsx

```diff
@@ -1,4 +1,7 @@
-import { type FC, type FormEvent, useReducer } from 'react';
+/* eslint-disable max-lines-per-function */
```

Comment:
@sgratch this file is part of your refactor?

Reply:
accidentally left disabing the rule in there since my whole file turned red. Removing it again